DeAceroDLabs/marketplace-frontend
# [DMP-609] Despliega las características

## What is done

[[DMP-609] Desplegar propiedades de producto](https://deacero.atlassian.net/browse/DMP-609)

## How is done

1. Adapt `Expandable component` to reutilize it.
2. Get Characteristics from the product detail based on the product ID.

## How to test

1. Run the project via `npm start`
3. Click on any product.
4. You should see the properties of the product listed below its name.
